scripts/tests/linux/log_line_processing.py

```diff
@@ -63,26 +63,30 @@ def _io_thread(self):
         err_wait = select.poll()
         err_wait.register(self.process.stderr, select.POLLIN | select.POLLHUP)
 
-        with open(self.output_path, "wt") as f:
+        with open(self.output_path, "wt", buffering=1) as f:
```

Comment:
Please add a comment on what buffering=1 means (it seems to be line buffered).

Reply:
Yes, I've added it in the latest commit.